tutorials/shaders/shader_reference/shading_language.rst

```diff
@@ -1079,6 +972,12 @@ Global uniforms are especially useful for environmental effects that affect many
 objects in a scene, like having foliage bend when the player is nearby, or having
 objects move with the wind.
 
+.. note:: *Global uniforms* are not the same as *global scope* for an individual
+    shader. While regular uniforms are defined outside of shader functions and are
+    therefore the global scope of the shader, global uniforms are global to all
+    shaders in the entire project (but within each shader, are also in the global
+    scope). Yes, it's a little confusing.
+
```

Comment:
For reviewers: this is the only really *new* content here, the rest is rearranging and rewording